### ISSUE TITLE
Improve Get Involved page usability

### ISSUE THREAD

**Issue by ethanjli** [apienc/dfp-portal#15]
The signup link should have actual text for "click here to sign up" for accessibility (e.g. with screen readers) and visual consistency, so that people actually know it's a link. It could also be cool if we added some photos here. We can expand the blurb to say that time commitment is up to the volunteer, training will be provided, and provide other helpful details.
